### PR TITLE
Remove NemID: final

### PR DESCRIPTION
**Pull request by nmoskaleva**
Remove all NemID references (+ some formatting changes).

### CODE REVIEW

File: src/pages/verify/e-ids/danish-mitid.mdx [hunk 1]
```diff
@@ -123,27 +123,24 @@ If in doubt, our recommendation would generally be to go with Substantial (`urn:
...
+**Prerequisites for ordering**
 
-  In order to apply for MitID in production on behalf of a company you must meet the basic requirements:
+In order to apply for MitID in production on behalf of a company you must meet the basic requirements:
 
-  - Your company must be registered in the EU and have an EU VAT Id.
-  - The person applying must have a NemID/MitID employee signature from the company in question, a socalled "medarbejdersignatur"
-  - You must have completed step 5 in the [Getting ready for production](/verify/guides/production) guide. You will need the production domain to complete the order for your client credentials.
+- Your company must be registered in the EU and have an EU VAT Id.
+- The person applying must have a MitID employee signature from the company in question, a socalled "medarbejdersignatur"
```
Comment (sgryt) [inline src/pages/verify/e-ids/danish-mitid.mdx:137]: -> The person applying must be enrolled in MitID Erhverv for your company, or be a signatory of the company.

File: src/pages/verify/e-ids/danish-mitid.mdx [hunk 2]
```diff
@@ -158,49 +155,46 @@ Please [see the UX requirements](#mitid-user-interface-requirements) to make sur
...
 1. Go to the [management dashboard](https://dashboard.criipto.com) and set the environment toggle at the top center to "PRODUCTION".
 2. In the "Identity sources" section, expand the "DK MitID" section
-3. A user with a NemID/MitID employee signature, a socalled "medarbejdersignatur" must click the button and sign in.
-3. Submit the details for your company. Note the following:
-  - The name to show in the MitID login box is the name entered in the "Company alias" box
-  - The "Domain prefix" is typically your company or brand name, e.g. `acme-corp`. Once this registration is completed this will be used to set up your MitID domain, in this case `acme-corp.mitid.dk`.
-4. The information is sent to Nets who will setup your MitID Domain.
-  - Expect this process to take 7-15 work days.
-  - Once your domain has been set up, you can complete the onboarding process (read below)
+3. A user with a MitID employee signature, a socalled "medarbejdersignatur" must click the button and sign in.
```
Comment (sgryt) [inline src/pages/verify/e-ids/danish-mitid.mdx:162]: with a MitID employee signature, a socalled "medarbejdersignatur"
->
enrolled in MitID Erhverv

File: src/pages/verify/e-ids/danish-mitid.mdx [hunk 3]
```diff
@@ -158,49 +155,46 @@ Please [see the UX requirements](#mitid-user-interface-requirements) to make sur
...
+- The name to show in the MitID login box is the name entered in the "Company alias" box
+- The "Domain prefix" is typically your company or brand name, e.g. `acme-corp`. Once this registration is completed this will be used to set up your MitID domain, in this case `acme-corp.mitid.dk`.
+5. The information is sent to Nets who will setup your MitID Domain.
+- Expect this process to take 7-15 work days.
+- Once your domain has been set up, you can complete the onboarding process (read below)
 
 #### Companies not registered in Denmark
 
-If you company is *not* registered in Denmark, meaning you don't have NemID/MitID employee signatures ("medarbejdersignatur"), please send a request for MitID onboarding to [support@criipto.com](mailto:support@criipto.com).
+If you company is *not* registered in Denmark, meaning you don't have MitID employee signatures ("medarbejdersignatur"), please send a request for MitID onboarding to [support@criipto.com](mailto:support@criipto.com).
```
Comment (sgryt) [inline src/pages/verify/e-ids/danish-mitid.mdx:172]: MitID employee signatures ("medarbejdersignatur")
->
MitID Erhverv

Comment (nmoskaleva) [reply src/pages/verify/e-ids/danish-mitid.mdx]: @sgryt all changed

**Pull request merged**